apps/condo/domains/contact/components/ContactsEditor/index.tsx

```diff
@@ -71,7 +71,7 @@ export const ContactsEditor: React.FC<IContactEditorProps> = (props) => {
     searchContacts(client, {
         organizationId: organization,
         propertyId: property ? property : undefined,
-        unitName: unitName ? unitName : undefined,
+        unitName: unitName ? unitName : null,
```

Comment:
I think, a case, when no `unitName` is provided, should be handled differently in some explicit way.
Currently, a `Contact` schema does't requires `unitName` to be set.
So, we potentially can have more than 1000 contacts without `unitName` and the same error will arise.
Maybe to not fetch contacts at all?
For example, when `unitName` is not set, call `setContacts([])`.